keyless/circuit-data/templates/helpers/hashtofield.circom

```diff
@@ -25,6 +26,8 @@ template CheckAre64BitLimbs(numLimbs) {
 // Hashes multiple bytes to one field element using a Poseidon hash
 // We hash the length of the input as well to prevent collisions
 // Currently does not work with greater than 64*31=1984 bytes
+//
+// Warning: `numBytes` cannot be 0.
```

Comment:
This is pretty self-explanatory, no? It doesn't really make sense to hash zero bytes

Reply:
but it seems many hash algorithms accept 0-byte input?

Reply:
I see, didn’t realize that 😅